ktxspec.adoc

```diff
@@ -866,6 +868,9 @@ that reads KTX files must not assume that value fields are NUL
 terminated). `keyAndValueByteLength` does not include the bytes in
 `<<_valuepadding,valuePadding>>`.
 
+`keyAndValueByteLength` must be at least 3, that is a 1 byte key plus
+its NUL terminator and the value's NUL terminator.
```

Comment:
At one point, we had a metadata entry without value (just key). Given that the spec allows non-string values, NUL terminator is optional. So the minimum possible length is 2.

Reply:
Oops! Fix in PR 105.